### PR TITLE
Feature leaderboard command

### PR DESCRIPTION
**Pull request by Kevnkkm**
Implements the leaderboard command that generates a table image. It can be sorted and has pages

### CODE REVIEW

File: cogs/leaderboard.py [hunk 1]
```diff
@@ -56,7 +41,7 @@ async def leaderboard(
         tabledata = {
             "Placement": placements,
             "Player": [player["name"] for player in data],
-            "Rank": [calcRank(player["mmr"]) for player in data],
+            "Rank": [getRankByMMR(player["mmr"]) for player in data],
```
Comment (probablyjassin) [inline cogs/leaderboard.py]: getRankByMMR() -> Rank Object
use Rank.name
getRankByMMR().name

**Comment by probablyjassin** [comment]
move to cogs/profiles pls

**Comment by Kevnkkm** [comment]
I got cooked medium rare

**Comment by probablyjassin** [comment]
@Kevnkkm sorting is still completely broken because you use mongodb's `.sort(sort.lower(), DESCENDING)` for sorting. But when trying to sort by "winrate", "wins" or "losses", that does not work because these not fields in the database.

File: cogs/profiles/leaderboard.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+            tabledata["Player"].append(player["name"])
+            tabledata["Rank"].append(getRankByMMR(player["mmr"]).name)
+            tabledata["MMR"].append(player["mmr"])
+            tabledata["Wins"].append(wins)
+            tabledata["Losses"].append(losses)
+            winrate = round(wins / (wins + losses) * 100, 2) if wins + losses > 0 else 0
+            tabledata["Winrate %"].append(winrate)
+
+        df = pd.DataFrame(tabledata).set_index("Placement")
+        # df = df.sort_values(by="MMR", ascending=False)
```
Comment (Kevnkkm) [inline cogs/profiles/leaderboard.py]: @probablyjassin I forgot to uncomment this and change `by="MMR"` to the sort Option from Discord 😓

**Pull request merged**